core/src/components/breadcrumbs/breadcrumbs.scss

```diff
@@ -0,0 +1,46 @@
+@import "./breadcrumbs.vars";
+
+// Breadcrumbs
+// --------------------------------------------------
+
+:host {
+  /**
+   * @prop --background: Background of the breadcrumbs
+   * @prop --color: Text color of the breadcrumbs
+   */
+  @include font-smoothing();
+  @include padding($breadcrumbs-padding-top, $breadcrumbs-padding-end, $breadcrumbs-padding-bottom, $breadcrumbs-padding-start);
```

Comment:
We chatted about this earlier, but we should adjust this padding when used in `ion-toolbar`. Optionally remove altogether in favor of having devs use the `ion-padding` class when used in `ion-content`.